subjects/ispowerof2.en.md

```diff
@@ -15,7 +15,7 @@ This program must print `true` if the given number is a power of 2, otherwise it
 ```console
 student@ubuntu:~/ispowerof2$ go build
 student@ubuntu:~/ispowerof2$ ./ispowerof2 2 | cat -e
-true$
```

Comment:
correct me if i'm wrong, but 2^x = 2
that means that x = 1 so it is power of 2, right?
http://www.tsm-resources.com/alists/pow2.html

Reply:
I see. I've just translated task differently. Thanks now i understand =)